spring-cloud-huawei-sample/swagger-demo/springcloud-provider/src/main/resources/application.yaml

```diff
@@ -15,4 +15,5 @@ spring:
         version: 0.0.1
         healthCheckInterval: 30
 
+withJavaChassis: true
```

Comment:
May be we can add common prefix for this project configuration. 
e.g.  ```spring.cloud.swagger.enableJavaChassisAdapter```

Reply:
change to spring.cloud.servicecomb.swagger.enableJavaChassisAdapter